src_js/components/sidebar/SidebarContent.tsx

```diff
@@ -62,13 +64,14 @@ function SitemapPage(props: {
     <a href={props.page.url} tabIndex={-1}>
       <details class={props.dedent ? '' : 'primer-spec-toc-sitemap-item'}>
         <summary
+          class={props.page.external ? 'primer-spec-toc-sitemap-external' : ''}
           role="link"
           onClick={(e) => {
             e.preventDefault();
             window.location.href = props.page.url;
           }}
         >
-          {title}
+          {props.page.external && <i class={IconType.EXTERNAL_LINK} />} {title}
```

Comment:
Nice! Couple of suggestions (feel free to push back / counter-propose your own ideas):

- Could we try matching the styling of Wikipedia? (Example: https://en.wikipedia.org/wiki/Victor_Borge#External_links)
  - I like how the icon is placed at the end. It also looks a bit smaller than the current icon.
  - Looks like Wikipedia uses a background image to display the external link icon. We don't have to do that, we can continue using FontAwesome as you proposed :)
- I'm divided as to whether to open the external link in a new tab.
  - To me, it feels intuitive that a link marked "external" is unrelated to the rest of the project spec — if I navigate to this external page, I won't have an easy way to come back to this spec.
  - But it looks like Wikipedia doesn't open external links in new tabs 😆

Reply:
> I like how the icon is placed at the end. It also looks a bit smaller than the current icon.

I agree!

> Looks like Wikipedia uses a background image to display the external link icon. We don't have to do that, we can continue using FontAwesome as you proposed :)

Ah, I used FontAwesome just to be consistent with the other icons. Both options are just decorative, since either way the image is rendered by CSS, right?

> I'm divided as to whether to open the external link in a new tab.

I wouldn't say I have strong feelings about this, but my impression is that it's generally preferable to open links in the same tab. Screen readers can have trouble with links that open in a new tab, and it's easier to navigate via the back bottom than via tabs on mobile devices.

Reply:
> Ah, I used FontAwesome just to be consistent with the other icons. Both options are just decorative, since either way the image is rendered by CSS, right?

Yeah I think either approach works fine. I suspect using a background image makes it easier to vertically-center the icon. But you're call :)

Tangent: Given that we are using an icon to indicate external links, I also wondered about accessibility for non-sighted users: Should we include some sort of label to indicate that these links are "external links" to screen readers? We can do this either via a `title` attribute, or by adding some text in a [`<span class="sr-only">`](https://primer.style/css/utilities/layout#screen-reader-only).

---

> but my impression is that it's generally preferable to open links in the same tab

It looks like your statement aligns with the "best practices". Even though it doesn't align with my intuitions (and I wonder how many people in general agree with this, looks like there are a lot of mixed opinions on the Internet), I think I can be okay with not opening external links in new tabs 😛